### ISSUE TITLE
Error when stop is invoked

### ISSUE THREAD

**Issue by jaime1988** [Burgestrand/Hallon#109]
Hi, 

The previous error is fixed thanks! But the stop problem remains, when I try to invoke stop, I receive that:

``` ruby
/Users/jaime/.rvm/gems/ruby-1.9.3-p125/gems/hallon-0.16.0/lib/hallon/player.rb:138:in `stop': OpenAL error: 40964 (remove buffers!) (RuntimeError)
    from /Users/jaime/.rvm/gems/ruby-1.9.3-p125/gems/hallon-0.16.0/lib/hallon/player.rb:138:in `block in status='
    from /Users/jaime/.rvm/gems/ruby-1.9.3-p125/gems/hallon-0.16.0/lib/hallon/audio_queue.rb:98:in `block in synchronize'
    from /Users/jaime/.rvm/rubies/ruby-1.9.3-p125/lib/ruby/1.9.1/monitor.rb:211:in `mon_synchronize'
    from /Users/jaime/.rvm/gems/ruby-1.9.3-p125/gems/hallon-0.16.0/lib/hallon/audio_queue.rb:98:in `synchronize'
    from /Users/jaime/.rvm/gems/ruby-1.9.3-p125/gems/hallon-0.16.0/lib/hallon/player.rb:128:in `status='
    from /Users/jaime/.rvm/gems/ruby-1.9.3-p125/gems/hallon-0.16.0/lib/hallon/player.rb:185:in `stop'
    from test.rb:100:in `<main>'
´´´
```

**Comment by Burgestrand** [comment]
I believe this happens when one tries to stop the player while music is playing (as opposed to pausing it). The bug's related to the OpenAL driver for Hallon. Will create a bug report in that repository and track it from there.

**Comment by Burgestrand** [comment]
I believe I found the issue (hard to be sure when it only happens sometimes). Released a new version of hallon-openal (v1.0.0). This is the only fix for this version bump, but the API is stable which is why it jumped up to v1.0.0 from v0.0.3.
